webdriver/pom.xml

```diff
@@ -37,7 +37,7 @@
     </distributionManagement>
 
     <properties>
-        <selenium.version>2.47.0</selenium.version>
+        <selenium.version>2.49.1</selenium.version>
```

Comment:
I know that there is issues with Selenium versions, but it's always the case. They release new browsers and new Selenium all the time. Updating the dependency is a different track.